modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -617,7 +618,8 @@ ONNXImporter::TensorInfo ONNXImporter::getBlobExtraInfo(const std::string& input
 }
 
 void ONNXImporter::addLayer(LayerParams& layerParams,
-                            const opencv_onnx::NodeProto& node_proto)
+                            const opencv_onnx::NodeProto& node_proto,
+                            int drop_inputs_since_index)
```

Comment:
I think more informative to rename such argument to `num_inputs`:
```
num_inputs = std::min(node_proto.input_size(), num_inputs);
```

Reply:
Okay, done.